### ISSUE TITLE
[Bug][Tabstrip] Tabstrip does not update the `:focus` state of the tabs on keyboard navigation

### ISSUE THREAD

**Issue by vveesseelliinnaa** [telerik/kendo-react#1777]
## I'm submitting a...
* Bug report 

## Current behavior
When the user navigates through the tab via the keyboard arrows, the focus state of the tabs is not updated and the focus-related styles are not applied, respectively.
![image](https://github.com/telerik/kendo-react/assets/6196386/a5c9badc-c873-48c1-8db2-e6c74eeb903b)

## Expected behavior
The focus state should be updated and the style of the selected tab should be the same like if the tab is clicked (refer the Angular Tabstrip).
![image](https://github.com/telerik/kendo-react/assets/6196386/4c2a17e3-d740-4f31-9ee2-c377b90fdc42)

## Minimal reproduction of the problem with instructions
1. Go to https://www.telerik.com/kendo-react-ui/components/layout/tabstrip/keyboard-navigation/
2. Click on the `Paris` tab
3. Press the right arrow button

Result: The selected tab is changed but the focus is still on the first one

Related to https://github.com/telerik/kendo-vue/issues/480

**Comment by silviyaboteva** [comment]
Fixed in `v8.1.0-develop.13`.

Working demo: https://stackblitz.com/edit/react-54izvz?file=app%2Fmain.tsx
